packages/image_picker/image_picker_platform_interface/test/new_method_channel_image_picker_test.dart

```diff
@@ -929,6 +929,23 @@ void main() {
         expect(response.file!.path, '/example/path');
       });
 
+      test('getLostData should successfully retrieve multiple files', () async {
+        picker.channel.setMockMethodCallHandler((MethodCall methodCall) async {
+          return <String, dynamic>{
+            'type': 'image',
+            'path': '/example/path1',
+            'pathList': ['/example/path0', '/example/path1'],
+          };
+        });
+        // ignore: deprecated_member_use_from_same_package
```

Comment:
Just to check, is this still needed? I'm not 100% sure on this, but I don't think these are deprecated.